cli/lightning-cli.c

```diff
@@ -170,10 +170,11 @@ static void add_input(char **cmd, const char *input,
 		tal_append_fmt(cmd, ", ");
 }
 
-static void
-exec_man (const char *page) {
-	execlp("man", "man", page, (char *)NULL);
-	err(1, "Running man command");
+static int
+spawn_man (const char *page) {
+	char command[sizeof("man") + strlen(page)];
+	snprintf(command, sizeof(command), "man %s", page);
+	return system(command);
```

Comment:
`lightning-cli help '; echo Hello world'`

It's nasty, but I think we need do this this properly (*UNTESTED CODE AHEAD!*):

```
/* If man command succeeds, exit silently.  Otherwise we return. */
static void spawn_man(const char *page) {
int status;

switch (fork()) {
case -1:
    err(1, "Forking man command");
case 0:
    /* child, run man command. */
    execlp("man", "man", page, (char *)NULL);
    err(1, "Running man command");
default:
    break;
}

wait(&status);
if (WIFEXITED(status) && WEXITSTATUS(status) == 0)
    exit(0);
}
```

Reply:
good call